internal/p2p/peermanager.go

```diff
@@ -405,6 +406,13 @@ func (m *PeerManager) Add(address NodeAddress) (bool, error) {
 	if ok {
 		return false, nil
 	}
+	if peer.Inactive {
+		return false, nil
+	}
+
+	// set the peer's mutable score to something non-zero so that
+	// peer's we've never seen aren't very low at start.
+	peer.MutableScore = DefaultMutablePeerScore
```

Comment:
If we're allowing negative values, setting this to 0 seems fine.

Reply:
I think the question is also more "should we persist the score" too.

Reply:
Sure, just saying that if we allow negatives, 0 is the middle of the range.